src/test/java/org/opensearch/knn/common/Constants.java

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.knn.common;
+
+public class Constants {
+    public static final String FILTER_FIELD = "filter";
```

Comment:
nit: FIELD_FILTER will provide better readability and can be grouped together

Reply:
Changed